src/olympia/abuse/views.py

```diff
@@ -267,45 +267,61 @@ def cinder_webhook(request):
     )
 
 
-def appeal(request, *, decision_id, abuse_report_id, **kwargs):
-    abuse_report = get_object_or_404(AbuseReport.objects, id=abuse_report_id)
+def appeal(request, *, abuse_report_id, decision_id, **kwargs):
+    cinder_job = get_object_or_404(
+        CinderJob.objects.exclude(
+            decision_action=CinderJob.DECISION_ACTIONS.NO_DECISION
```

Comment:
we probably want to exclude DECISION_ACTIONS.AMO_ESCALATE_ADDON too (can use `UNRESOLVED`)

Reply:
It would fail later anyway but I agree.

Any suggestion that reads better than this ? :

```python
appealable_decisions = list(
    CinderJob.DECISION_ACTIONS.APPEALABLE_BY_AUTHOR.values.keys()
) + list(CinderJob.DECISION_ACTIONS.APPEALABLE_BY_REPORTER.values.keys())
cinder_job = get_object_or_404(
    CinderJob.objects.exclude(decision_action__in=appealable_decisions),
    decision_id=decision_id,
)
```